Lib/asyncio/tasks.py

```diff
@@ -465,11 +466,15 @@ def _on_completion(f):
         yield _wait_for_one()
 
 
-@coroutine
-def sleep(delay, result=None, *, loop=None):
+@types.coroutine
+def _sleep0():
+    yield
```

Comment:
Need a comment here explaining why we have this hack.

Reply:
Done